ClangSharpPInvokeGenerator/CursorVisitor.cs

```diff
@@ -258,7 +268,7 @@ public CXChildVisitResult VisitTranslationUnit(CXCursor cursor, CXCursor parent,
         {
             Debug.Assert(parent.Kind == CXCursorKind.CXCursor_TranslationUnit);
 
-            if (cursor.Location.IsInSystemHeader || !cursor.Location.IsFromMainFile)
+            if (!cursor.Location.IsFromMainFile)
```

Comment:
Changed this to just check if we aren't in the main file; which ultimately allows users to generate bindings for system headers as well.